components/user-mgt/org.wso2.carbon.user.mgt.ui/pom.xml

```diff
@@ -142,6 +142,7 @@
                             org.wso2.carbon.user.mgt.ui.*;version="${carbon.identity.package.export.version}",
                             org.wso2.carbon.userstore.ui.*,
                          </Export-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Seems the lines are not properly formatted

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/3364/commits/f410d5f7369aeb3974e6843fa66a74c1aa25c542.